src/liballoc/boxed.rs

```diff
@@ -699,6 +700,13 @@ impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<Box<U>> for Box<T> {}
 #[unstable(feature = "dispatch_from_dyn", issue = "0")]
 impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<Box<U>> for Box<T> {}
 
+#[unstable(feature = "boxed_slice_from_iter", issue = "0")]
```

Comment:
The unstable attribute has no effect since impls are insta-stable so you can remove this line.